backend/test/model_test/invite_user_adm_test.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+"""Invite User Adm Test."""
+
+from .. import mocks
+from ..test_base import TestBase
+
+from models.institution import Institution
+from models.invite_user_adm import InviteUserAdm
+from models.user import User
+
+from custom_exceptions.notAuthorizedException import NotAuthorizedException
+
+
+class InviteUserAdmTest(TestBase):
+    """Test invite user adm model."""
+
+    @classmethod
+    def setUp(cls):
+        """Provide the base for the tests."""
+        cls.test = cls.testbed.Testbed()
+        cls.test.activate()
+        cls.policy = cls.datastore.PseudoRandomHRConsistencyPolicy(
+            probability=1)
+        cls.test.init_datastore_v3_stub(consistency_policy=cls.policy)
+        cls.test.init_memcache_stub()
+    
+    def test_create(self):
+        """Test create new invite."""
+        institution = mocks.create_institution()
+        admin = mocks.create_user()
+        new_admin = mocks.create_user()
+
+        institution.admin = admin.key
+        institution.members = [admin.key, new_admin.key]
```

Comment:
Never change the model outside the model, we have functions to do that, like add_member

Reply:
Done